DesignPattern/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace DesignPattern
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            WheatPizza wheatPizza;
+            WhitePizza whitePizza;
+
+            WheatPizzaBuilder wheatPizzaBuilder = new WheatPizzaBuilder();
```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.